git_machete/git_operations.py

```diff
@@ -167,9 +167,13 @@ def fetch_remote(self, remote: str) -> None:
         if remote not in self.__fetch_done_for:
             self.run_git("fetch", remote)
             self.__fetch_done_for.add(remote)
+            # TODO (#237): flush, remote_branches_cached and others
```

Comment:
For clarity... what is supposed to happen for these TODOs before the PR is merged? 🤔

Reply:
They're to be deleted - whole. I don't think it makes sense to keep them as a comment or smth like that. I created them for myself as a debug duck and for reviewers to help them with the understanding of the decisiones. After the review is completed/initially approved they no longer have any purpose.